### DISCUSSION TITLE
hit smallint limit on Prefix table

### DISCUSSION THREAD

**Discussion by andy-shady-org** [netbox-community/netbox#10276] [General]
Hi,

Ive hit 'smallint' limit on ipam_prefix table when trying to import 43k+ prefixes.
Ive tried setting the _depth column to bigint, but that didnt fix it. I've also tried clearing the cache, and reimporting the dataset.
Neither fixed the issue.

The issue seems to happen when saving a prefix, not creating one.

Any help would be great.

**Comment by kkthxbye-code** [comment]
Could you provide a full stacktrace?

**Reply by andy-shady-org** [reply]
can you please tell me how to do that?

**Reply by kkthxbye-code** [reply]
I think it should log to the journal if you follow the default install. So something like `journalctl -eu netbox`.

**Reply by andy-shady-org** [reply]
There is nothing in that output relating to this issue Im afraid.

**Reply by kkthxbye-code** [reply]
I'm not sure what the default logging config is when following the install tutorial, as I only use docker, but you can configure it by reading here: https://docs.netbox.dev/en/stable/configuration/system/#logging

Alternatively you can enable the `DEBUG` setting and trigger the issue again, and you should get debug output including a stacktrace.

**Reply by andy-shady-org** [reply]
Yes, I have debug turned on, Im rerunning the import on a fresh db, so Ill update this when its finished

**Reply by andy-shady-org** [reply]
I think Ive found the issue, there is a vlan id of 99999 in the dataset Im importing from Infoblox, its this insert that fails and creates the smallint issue.

Sorry for the inconvience :(

**Reply by jeremystretch** [reply]
No worries, glad you figured it out. Thanks for following up!
